caluma-demo/app/templates/documents/edit.hbs

```diff
@@ -1,3 +1,6 @@
 {{#if model}}
   {{cf-content documentId=model}}
+  <LinkTo @route="documents">
+    <button class="uk-button uk-button-primary">{{t "forms.cancel" }}</button>
+  </LinkTo>
```

Comment:
Just add `@tagName="button"` and the classes to the link-to helper

Reply:
Done.